apps/openmw/mwworld/cellpreloader.cpp

```diff
@@ -470,10 +461,25 @@ namespace MWWorld
         return true;
     }
 
+    void CellPreloader::abortTerrainPreloadExcept(const CellPreloader::PositionCellGrid *exceptPos)
```

Comment:
This function can be left in the place where it was to avoid enlarging the diff and making it harder to read.